global.json

```diff
@@ -1,5 +1,5 @@
 {
   "sdk": {
-    "version": "2.2.105"
+    "version": "3.1.100"
```

Comment:
[3.1.101](https://github.com/dotnet/core/blob/master/release-notes/3.1/3.1.1/3.1.1.md) is the latest version now